DuckDuckGo/RemoteMessagingSurveyURLBuilder.swift

```diff
@@ -17,18 +17,13 @@
 //  limitations under the License.
 //
 
-#if NETWORK_PROTECTION
-
 import Foundation
 import BrowserServicesKit
+import RemoteMessaging
 import Core
 import Common
 
-protocol SurveyURLBuilder {
-    func addSurveyParameters(to url: URL) -> URL
-}
-
-struct DefaultSurveyURLBuilder: SurveyURLBuilder {
+struct DefaultRemoteMessagingSurveyURLBuilder: RemoteMessagingSurveyActionMapping {
 
     enum SurveyURLParameters: String, CaseIterable {
```

Comment:
I think this enum can be safely deleted?

Reply:
Yes indeed, good catch - thanks!